products/db-checkbook/build_scripts/build.py

```diff
@@ -6,31 +6,36 @@
 
 from . import LIB_DIR, OUTPUT_DIR, SQL_QUERY_DIR
 
-DB_URL = 'sqlite:///checkbook.db'
+DB_URL = "sqlite:///checkbook.db"
```

Comment:
this is purely a comment  but reading [the current Black style documentation](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html) is so thorough it's wild - never really thought about the aesthetic merits of favoring double quotes over single quotes for strings lol

Reply:
If you want a rollercoaster of a discussion on it

https://github.com/psf/black/issues/118

Reply:
definitely thorough, imagine trying to make so many devs happy with a common style and (pretty much) succeeding!